### PR TITLE
[Beta] fix: gutter overlap with sandpack code on mobile

### LINKED ISSUE #5169: [Beta] Bug: Gutter text overlaps with sandpack code on mobile
Gutter text overlaps with code in the sandpack while scrolling horizontally on mobile devices.

https://user-images.githubusercontent.com/42532987/194760640-dc1dc914-b3e5-4c54-868f-922e8256c0d7.mp4

### PR DESCRIPTION
**Pull request by shanpriyan**
Fix #5169 

**After fix:**

https://user-images.githubusercontent.com/42532987/194761342-9fda6702-a0cd-43ec-9b74-ac1219bc2a46.mp4

### CODE REVIEW

**Review by harish-sethuraman — APPROVED** [review]
LGTM thanks for sending a PR

**Comment by gaearon** [comment]
This breaks the left border of the focus ring, which you get by pressing Escape when inside the code editor:

Screenshot 2022-10-09 at 23 57 43

Would you mind looking if there's some way to adjust that fixes both problems?

**Comment by shanpriyan** [comment]
Can we move the horizontal scrollbar just to code mirror content, similar to vs code editor?


## VS code editor
image

## Our app - After moving scrollbar to code content
image

**Comment by shanpriyan** [comment]
@gaearon please share your thoughts on this
>Can we keep the horizontal scrollbar only to code mirror content, similar to vs code editor?

**Comment by gaearon** [comment]
that sounds fine to me.

File: beta/src/styles/sandpack.css [hunk 1]
```diff
@@ -211,8 +211,14 @@ html.dark .sp-wrapper {
   -webkit-font-smoothing: auto; /* Improve the legibility */
 }
 
+.sandpack .sp-code-editor .cm-content {
+  overflow-x: auto;
```
Comment (harish-sethuraman) [inline beta/src/styles/sandpack.css:215]: `auto` keeps the scrollbar visible always. I guess scroll would be better.
```suggestion
      overflow-x: scroll;
```

**Comment by shanpriyan** [comment]
### Fixed gutter overlap
image

### Fixed focus ring
image

### Horizontal scrollbar 
|  Before fix | After fix |
| ------- | ------ |
| image|image|

**Comment by gaearon** [comment]
let's give this a try. thank you!

**Pull request merged**